index.html

```diff
@@ -320,12 +327,12 @@ <h4>Multikey</h4>
 
           <p>
 The `publicKeyMultibase` property represents a Multibase-encoded Multikey
-expression of a Dilithium public key.
-<span class="issue">Specify acceptable Dilithium key sizes</span>
+expression of a ML-DSA public key.
+<span class="issue">Specify acceptable ML-DSA key sizes</span>
```

Comment:
ML-DSA-44 public key size is 1312 bytes, private key size is 2528 bytes.

Reply:
Great, these values are already there. Removing this issue